rholang/src/test/scala/coop/rchain/rholang/interpreter/NormalizeTest.scala

```diff
@@ -646,6 +649,19 @@ class ProcMatcherSpec extends FlatSpec with Matchers {
     )
     result.knownFree should be(inputs.knownFree)
   }
+
+  it should "handle peek" in {
+    val basicInput = ParBuilder[Coeval]
+      .buildAST(
+        new StringReader("""for ( x, y <<- @Nil ) { x!(*y) }""")
+      )
+      .value()
+
+    val result = ProcNormalizeMatcher.normalizeMatch[Coeval](basicInput, inputs).value
+    result.par.receives.head.peek shouldBe true
+
```

Comment:
much better, altough this empty line is redundant :)

Reply:
:)